doc/specs/wiki.md

```diff
@@ -0,0 +1,348 @@
+# Wiki Specs
+
+## Table of contents
+
+1. Motivation
+2. Business rules
+3. Use cases
+4. Data model
+
+## Motivation
+
+>The site contains a variety of users with a vast wealth of knowledge. The wiki is a place where users can put their knowledge to work, and write articles for users to use. These may be supplementary to other documents on the site, or standalone documents such as guides.
+
+## Business rules
+
+> * Articles must be able to be read/write locked based on user class.
```

Comment:
idem. I think this is a result of my 'example' being a quote, but that was a quote because it was just an example.... I see how this was confusing :P